auth/src/main/res/values-v26/styles.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources
+    xmlns:tools="http://schemas.android.com/tools"
+    tools:ignore="ResourceName">
+
+    <style name="FirebaseUI.TextInputEditText.EmailField">
+        <item name="android:autofillHints">emailAddress</item>
```

Comment:
I think that's a good idea, when smartlock is enabled disable other autofills.  That's the path of least annoyance.